app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  color: #222222;
+  font-family: sans-serif;
+  font-size: 1.6rem;
+  line-height: 2.5rem;
```

Comment:
Prefer line-heights to be unitless values so that they always return the value that you expect.

https://github.com/StemboltHQ/information/blob/master/sass_styleguide.markdown